ios/Podfile

```diff
@@ -28,7 +40,12 @@ abstract_target 'RNTemplate' do
   end
 
   post_install do |installer|
-    react_native_post_install(installer)
+    react_native_post_install(
+      installer,
+      # Set `mac_catalyst_enabled` to `true` in order to apply patches
+      # necessary for Mac Catalyst builds
+      :mac_catalyst_enabled => false
+    )
 
     installer.aggregate_targets.each do |aggregate_target|
```

Comment:
we should try removing this block and see if it still works on M1 machines, because it was a temporal workaround that should be fixed now